dotnet/message-service/Startup.cs

```diff
@@ -27,10 +28,15 @@ public void ConfigureServices(IServiceCollection services)
                 c.SwaggerDoc("v1", new OpenApiInfo {Title = "message_service", Version = "v1"});
             });
 
-            services.UseHoneycomb(Configuration);
-
-            var multiplexer = ConnectionMultiplexer.Connect("localhost");
+            var redisConfig = Environment.GetEnvironmentVariable("REDIS_URL");
+            if (redisConfig == null)
+            {
+                redisConfig = "localhost";
+            }
+            var multiplexer = ConnectionMultiplexer.Connect(redisConfig);
             services.AddSingleton<IConnectionMultiplexer>(multiplexer);
+
+            services.UseHoneycomb(Configuration);

```

Comment:
Need to `.UseHoneycomb()` _after_ the Redis connection multiplexer is added to the service or else our autodetection of a Redis connection won't detect it.

Reply:
Just tested that on main, with previous setup (add redis, then add honeycomb), and seeing redis instrumentation. I don't think order matters

Reply:
Hunh. I _thought_ it was an ordering problem when I wasn't seeing Redis spans, but I was doing a lot of things. I'll try it again in the previous order!

Reply:
As you said, it worked when I put things back to the original order (UseHoneycomb then configure Redis). Updated this branch back to the original order to show that off. :sparkles: